storage/postgres.sql

```diff
@@ -44,6 +44,12 @@ create table if not exists users_games
     PRIMARY KEY (user_id, game_id)
 );
 
+create table if not exists game_to_map
+(
+    game_id bigserial,
```

Comment:
It would be really great, I have no idea like that without testing queries of the database.
We really need to move the SQL to SQL file for the readability, It will help us a lot I think @denverquane

Reply:
Yes, I will add that in. I was not terribly familiar with that pattern so I didn't include it while testing the feature. I will read up on it and make sure it gets tested as well.